### PR TITLE
feat: deployment support for testnet and mainnet 

### PR DESCRIPTION
**Pull request by negar-abbasi**

### CODE REVIEW

File: src/localnet.ts [hunk 1]
```diff
@@ -11,6 +11,14 @@ export async function isLocalNet(algod: Algodv2): Promise<boolean> {
 
   return params.genesisID === 'devnet-v1' || params.genesisID === 'sandnet-v1' || params.genesisID === 'dockernet-v1'
 }
+export async function isTestNet(algod: Algodv2): Promise<boolean> {
```
Comment (daniel-makerx) [inline src/localnet.ts:14]: These two functions should be moved to `src/network-client.ts`

File: src/types/app.ts [hunk 2]
```diff
@@ -269,6 +269,8 @@ export enum OnUpdate {
   UpdateApp,
   /** Delete the app and create a new one in its place */
   ReplaceApp,
+  /** Append the app */
```
Comment (daniel-makerx) [inline src/types/app.ts:272]:
```suggestion
  /** Create a new app */
```

File: src/types/app.ts [hunk 3]
```diff
@@ -277,6 +279,8 @@ export enum OnSchemaBreak {
   Fail,
   /** Delete the app and create a new one in its place */
   ReplaceApp,
+  /** Append the app */
```
Comment (daniel-makerx) [inline src/types/app.ts:282]:
```suggestion
  /** Create a new app */
```

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -21,6 +21,7 @@ jobs:
       pre-test-script: |
         pipx install algokit
         algokit localnet start
+        npx --yes wait-on tcp:4001 -t 30000
```
Comment (robdmoore) [inline .github/workflows/release.yml:24]: Given the recent changes in algokit cli I *think* we don't need this anymore

**Review by robdmoore — APPROVED** [review]

**Pull request merged**